### PR TITLE
proposed copy edits to rustdoc chapter

### PR DESCRIPTION
**Pull request by zackmdavis**
(updating a link, and making commands more literally copy-pasteable)

### CODE REVIEW

**Comment by mark-i-m** [comment]
So doxidize is officially the future rustdoc 2?

**Comment by zackmdavis** [comment]
@steveklabnik ^^ What, if anything, should be said about future-rustdoc?

**Comment by steveklabnik** [comment]
I think, for now, we should simply remove the references to this project. It's not in a state where we want people to poke at it, and it's not going to be until the fall, at least.

File: src/rustdoc.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Rustdoc actually uses the rustc internals directly. It lives in-tree with the
 compiler and standard library. This chapter is about how it works. (A new
 implementation is also [under way], though).
```
Comment (mark-i-m) [inline src/rustdoc.md]: Could you change this to something more vague (as per https://github.com/rust-lang-nursery/rustc-guide/pull/150#issuecomment-395783504)

Perhaps something like "There have been some attempts to rewrite rustdoc, but they are in early stages".

**Comment by mark-i-m** [comment]
Ping @zackmdavis

**Comment by zackmdavis** [comment]
@mark-i-m pong (just scrapped the sentence entirely)

**Comment by mark-i-m** [comment]
Thanks!

**Pull request merged**